src/pages/index.js

```diff
@@ -135,20 +314,78 @@ document.querySelectorAll(".modal").forEach((modal) => {
 
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = editModalNameInput.value;
-  profileDescription.textContent = editModalDescriptionInput.value;
-  closeModal(editModal);
+  const submitButton = editFormElement.querySelector(".modal__submit-btn");
+
+  setSubmitButtonState(submitButton, true, "Saving...", "Save");
+
+  api
+    .editUserInfo({
+      name: editModalNameInput.value,
+      about: editModalDescriptionInput.value,
+    })
+    .then((data) => {
+      profileName.textContent = data.name;
+      profileDescription.textContent = data.about;
+      editFormElement.reset();
+      closeModal(editModal);
+      resetValidation(editFormElement, settings);
```

Comment:
`resetValidation` is not needed after **submission** because there can't be any validation errors if the submission is successful